addons/medical_feedback/functions/fnc_playInjuredSound.sqf

```diff
@@ -28,6 +28,15 @@ if (!local _unit) exitWith {
 };
 if !(_unit call EFUNC(common,isAwake)) exitWith {};
 
+// Limit network traffic by only sending the event to players who can potentially hear it
+private _distance = if (_type == "hit") then {
+    [50, 60, 70] select _severity;
+} else {
+    [10, 15, 20] select _severity;
+};
+private _targets = allPlayers inAreaArray [getPosWorld _unit, _distance, _distance];
```

Comment:
`allPlayers` is a really shitty command and is often not synced correctly
it can take minutes after connecting for someone to show up in the list
tbh, that's probably ok as groans aren't super critical

Reply:
This should handle z axis, but so rare it's not really needed
```suggestion
private _targets = allPlayers inAreaArray [getPosWorld _unit, _distance, _distance, 0, false, _distance];
```